mlir/test/rocmlir-driver/auto_e2e/xdlops/lit.local.cfg

```diff
@@ -1,2 +1,2 @@
-if not config.enable_rock_driver_e2e_test or config.no_AMD_GPU or not config.xdlops:
+if not config.enable_rock_driver_e2e_test or config.no_AMD_GPU or "-mfma=on" not in config.features:
```

Comment:
For this, I'd suggest something different.
Specifically, what I'd go for is to, over in `lit.site.cfg.py.in`, look through `%arch` to check if it contains `gfx908`, `gfx90a`, or so on, and, if it does, set `config.arch_has_mfma = True`